### ISSUE TITLE
Add dependency check for ncat

### ISSUE THREAD

**Issue by popxunga** [asbru-cm/asbru-cm#643]
Hi ...
Once the ncat command is used for proxy connections then shouldn't we add a dependency that ensures the installation of the package that contains the ncat command ?
In the EL7 and EL8 the ncat command is provided by the nmap-ncat package.
Thanks.

**Comment by gfrenoy** [comment]
Good point.

I can probably add it to the "Recommends" section.

So this is the package: https://pkgs.org/download/nmap-ncat

And for Debian, we need the [`ncat`](https://packages.debian.org/buster/ncat) package

**Comment by gfrenoy** [comment]
Done.

But by looking at this I'm wondering if we cannot improve a bit the dependencies ; I'll open a new issue for that.

**Comment by popxunga** [comment]
Hi @gfrenoy 
The "Recommends" addition to the Asbru spec file makes the build fail on EL7.
If you look into the loki repo you will see that there's no EL7 build after this modification.

**Comment by gfrenoy** [comment]
Thanks for the check !  I've move it to "Depends" ; which actually makes more sense since some features will not work without it so it is more than a strong requirement...

**Comment by Frantzd** [comment]
Hello,

There is no package "ncat" in Ubuntu 18.04, it is provided by "nmap", so that dependencies are broken :-( :

```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
You might want to run 'apt --fix-broken install' to correct these.
The following packages have unmet dependencies:
 asbru-cm : Depends: ncat but it is not installable
E: Unmet dependencies. Try 'apt --fix-broken install' with no packages (or specify a solution).
```

**Comment by gfrenoy** [comment]
This has been fixed in 6.3.2, see #776

**Comment by Frantzd** [comment]
ok, great :-)
